dcrtimed/backend/backend.go

```diff
@@ -110,6 +110,14 @@ type FsckOptions struct {
 	File string // Path for results file
 }
 
+// GetBalanceResult contains information account balance
+// info for wallet used by the backend.
+type GetBalanceResult struct {
```

Comment:
This needs JSON magic to name the fields.

Reply:
Fixed. This type is purely backend stuff, and not serialized to JSON. Is there more work to do for this PR with respect to serialization?

Reply:
I missed that part. I thought they were being serialized. The rule is when serialized add attributes. So you can disregard my earlier comment.